### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by pollfly**
* Correct clearml-agent setup return
* Add ``` tick for snippets - code spacing was a bit off
* remove demoapp references

### CODE REVIEW

File: docs/clearml_agent.md [hunk 1]
```diff
@@ -91,37 +96,41 @@ Install ClearML Agent as a system Python package and not in a Python virtual env
...
 1. The wizard responds with your configuration:
-    
-        CLEARML Hosts configuration:
-        Web App: https://app.community.clear.ml
-        API: https://demoapi.clearml.allegro.ai
-        File Store: https://demofiles.clearml.allegro.ai
+   ```
+   CLEARML Hosts configuration:
+   Web App: https://app.community.clear.ml
+   API: https://api.clearml.ml
```
Comment (ainoam) [inline docs/clearml_agent.md:102]: @pollfly api.clear.ml

File: docs/clearml_agent.md [hunk 2]
```diff
@@ -91,37 +96,41 @@ Install ClearML Agent as a system Python package and not in a Python virtual env
...
 
 1. The wizard responds with your configuration:
-    
-        CLEARML Hosts configuration:
-        Web App: https://app.community.clear.ml
-        API: https://demoapi.clearml.allegro.ai
-        File Store: https://demofiles.clearml.allegro.ai
+   ```
+   CLEARML Hosts configuration:
+   Web App: https://app.community.clear.ml
```
Comment (ainoam) [inline docs/clearml_agent.md:101]: app.clear.ml

File: docs/guides/ui/building_leader_board.md [hunk 3]
```diff
@@ -18,7 +18,6 @@ Bookmark the leaderboard in the browser to reuse and share the leaderboard URL w
 For this tutorial, use one of the following as a project:
```
Comment (ainoam) [inline docs/guides/ui/building_leader_board.md:18]: @pollfly Is it "on of the following" now there's only one left?

Comment (pollfly) [reply docs/guides/ui/building_leader_board.md:18]: I see two - using your own project or executing example scripts (... to create your own project)

**Pull request merged**